std_msgs/Header header
int32 defenders_count
int32[] defenders_sectors
# position in grid, in ENU
geometry_msgs/Point32[] defenders_position
